assets/css/tasks/_blog-tasks.css

```diff
@@ -0,0 +1,133 @@
+/* POSTS PAGE */
+li:first-child {
+    margin-top: 0;
+     /* The :first-child CSS pseudo-class represents the first element among a group of sibling elements. */
```

Comment:
After adding value for css property you can delete a comment. It doesn't bring useful information about styled element anymore, because you have it in you code :)